config.toml

```diff
@@ -35,6 +35,18 @@ disableLanguages = ["hi", "no"]
     noClasses = true
     style = "emacs"
     tabWidth = 4
+  [markup.tableOfContents]
+    endLevel = 2
+    ordered = false
+    startLevel = 2
+  [markup.goldmark.parser]
+      attribute = true
+      autoHeadingID = true
+      autoHeadingIDType = "blackfriday"
```

Comment:
For future reviewers: this attribute setting is a smart choice in the landscape of Hugo 0.62.2.

https://github.com/gohugoio/hugo/issues/6616